server/main.cpp

```diff
@@ -48,20 +48,18 @@ void addNewPlayer(const std::string &name, websocketpp::connection_hdl hdl)
 
 void startGame() {
 	gameState.phase = GamePhase::GAME;
-	for (auto &p : gameState.players)
-	{
-		auto con = websocket_server.get_con_from_hdl(p->conn_hdl);
-		con->set_message_handler(&gameOnMessage);
-	}
+	dfws::SetOnMessage(&gameOnMessage);
 	new std::thread(gameThread); // leak the shit out of it yooo
 }
 
-void on_message(websocketpp::connection_hdl hdl, const server::message_ptr& msg)
+void mainOnMessage(dfws::Handle hdl, const std::string& payload)
 {
-	std::cout << "on_message\n";
-	std::cout << "message from " << (uint64_t)hdl.lock().get() << "\n";
+	std::cout << "mainOnMessage\n";
+	std::cout << "message from " << hdl << "\n";
```

Comment:
Are these left intentionally or debug?

Reply:
Actually both :P these messages help me to debug stuff sometimes. However they are not crucial, they are just for me. Removed.